components/ui/CardTitle.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+defineProps<{
+    forId: string;
+    text: string;
+}>();
+</script>
+
+<template>
```

Comment:
pretty straight forward for me. dont create this component. 
> use label directly

Reply:
Already remove unnecessary components